src/version/common/Topbar/fragments/UserPanelButton.js

```diff
@@ -35,50 +35,59 @@ const styles = {
  */
 class UserPanelButton extends Component {
 
+    constructor(props) {
+        super(props);
+        this.button = React.createRef();
+    }
+
     state = {
         anchorEl: null,
+        isOpen: false,
     };
 
-    handleMenu = event => {
-        this.setState({ anchorEl: event.currentTarget });
+    handleMenu = () => {
+        this.setState(state => ({
+            anchorEl: this.button.current,
+            isOpen: !state.isOpen,
+        }));
     };
 
     handleClose = () => {
-        this.setState({ anchorEl: null });
+        this.setState(state => ({
+            anchorEl: null,
+            isOpen: !state.isOpen,
+        }));
     };
 
-    componentDidMount() {
-        this.props.getUserInfo();
-    }
-
     render() {
-        const { classes, logout, userInfo } = this.props;
-        const { anchorEl } = this.state;
-        const isTopbarMenuOpen = Boolean(anchorEl);
+        const { classes } = this.props;
+        const { isOpen, anchorEl } = this.state;
         return (
-            <div className={classes.rightBlockItem}>
-                <IconButton
-                    id="icon-profile"
-                    className={classes.rightBlockButton}
-                    aria-owns={isTopbarMenuOpen ? 'menu-appbar' : undefined}
-                    aria-haspopup="true"
-                    onClick={this.handleMenu}
-                    color="inherit" >
-                    <PersonIcon />
-                </IconButton>
+            <div className={classes.rightBlockItem} ref={this.button}>
+                <Tooltip title="User panel">
+                    <IconButton
+                        id="icon-profile"
+                        className={classes.rightBlockButton}
+                        aria-owns={isOpen ? 'menu-appbar' : undefined}
+                        aria-haspopup="true"
+                        onClick={this.handleMenu.bind(this)}
```

Comment:
no need for the bind here, as you defined `handleMenu` using the arrow function in the class properties.